src/install/updates/update_v0.13.x_v0.x.x.php

```diff
@@ -10,4 +12,9 @@
 require_once(dirname(__FILE__) . "/../../inc/defines/config.php");
 require_once(dirname(__FILE__) . "/../../inc/defines/log.php");
 
+if (!isset($PRESENT["v0.13.x_hash_length"])) {
+  Factory::getAgentFactory()->getDB()->query("ALTER TABLE `Hash` MODIFY `hash` MEDIUMTEXT NOT NULL;");
```

Comment:
What if the database type is set to longtext? This change would modify that and thus cause issue when upgrading. Maybe add a check to this to check if the type is TEXT.